js/gateio.js

```diff
@@ -2941,7 +2941,7 @@ module.exports = class gateio extends Exchange {
         const status = this.parseOrderStatus (rawStatus);
         return this.safeOrder ({
             'id': this.safeString (order, 'id'),
-            'clientOrderId': this.safeString (order, 'text'),
+            'clientOrderId': undefined,
```

Comment:
@samgermain Do they have any other field that defines the clientOrderId? Is using `text` incorrect? Or is it really `undefined` for all orders ?

Reply:
I put it back in, this is what the gateio api defines it as 

![](https://i.imgur.com/eBe7ts5.png)